addons/website/static/src/client_actions/configurator/configurator.js

```diff
@@ -447,17 +459,40 @@ Object.assign(FeaturesSelectionScreen, {
     template: 'website.Configurator.FeatureSelection',
 });
 
+
 class ThemeSelectionScreen extends ApplyConfiguratorScreen {
     setup() {
+        const uiService = useService('ui');
```

Comment:
move it 3 lines below next to useService('orm') and go with the same notation: this.uiService instead of const? To be consistent and not have useXXX splitted here and there